docs/js/ra-doc-exec.js

```diff
@@ -1,5 +1,43 @@
-/* global Prism, prettier, prettierPlugins */
-var allMenus, navLinks, versionsLinks;
+/* global Prism */
+import { transpileModule } from 'https://esm.sh/typescript@5.7.3';
+import * as prettier from 'https://esm.sh/prettier@3.5.1/standalone';
+import * as babel from 'https://esm.sh/prettier@3.5.1/plugins/babel';
+import * as estree from 'https://esm.sh/prettier@3.5.1/plugins/estree';
```

Comment:
Why did you add this plugin?

Reply:
We had it before and we still need it so that prettier understand TS

Reply:
Correction: it's actually unnecessary